### ISSUE TITLE
Incorrect Jackson Version Naming 212 Vs 213?

### ISSUE THREAD

**Issue by maxstreese** [circe/circe-jackson#306]
Hey, just a quick question about [this part]( https://github.com/circe/circe-jackson/blob/0fc69cedc6ab59220eb515e9281b6ceb8358f180/build.sbt#L165) of the `build.sbt`: Shouldn't the Circe project be name `circe-jackson213` instead of `circe-jackson212` or am I missing something here? In case I am correct just let me know and I will make a tiny PR - thanks! If I am not I am curious to know what I am missing :sweat_smile:

**Comment by travisbrown** [comment]
Yes, not sure what happened there. I think we need to be cross-publishing both for Jackson 2.12 and 2.13. A PR would be much appreciated, thanks.
